api/surveys/short/questions.yaml

```diff
@@ -10,4 +10,7 @@ questions:
   - Hamburg
   - Cologne
   - Geneva
-  - Oslo
\ No newline at end of file
+  - Oslo
+- type: email-text
```

Comment:
I'd prefer to call it just "email".
Also, we need to add docs.
Can you add it to "simple" instead of "short"?

Reply:
have changed email-text to email and added docs for the same, have moved the question from short to simple survey